common/app/common/controllers/SunriseController.java

```diff
@@ -32,7 +34,7 @@
     protected SunriseController(final ControllerDependency controllerDependency) {
         super(controllerDependency.sphere());
         // TODO Fill it properly
-        final UserContext userContext = UserContext.of(GERMAN, emptyList(), DE, ZoneId.of("Europe/Berlin"));
+        final UserContext userContext = UserContext.of(DE, GERMAN, emptyList(), ZoneId.of("Europe/Berlin"), Reference.of(Zone.typeId(), "f77ddfd4-af5b-471a-89c5-9a40d8a7ab88"));
```

Comment:
It is hardcoded just because we are not going to implement the functionality that changes (most of) this information in the MVP, so so far it stays there hardcoded. So we don't forget, I've created an issue: https://github.com/sphereio/sphere-sunrise/issues/155